src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -342,6 +359,8 @@ class FunSpec private constructor(builder: Builder) {
     internal const val CONSTRUCTOR = "constructor()"
     internal const val GETTER = "get()"
     internal const val SETTER = "set()"
+    internal const val RETURN = "return "
+    @JvmField internal val RETURN_CODE_BLOCK = CodeBlock.of(RETURN)
```

Comment:
Makes sense. There's a number of places where this technique is used on `internal` members, I guess some cleanup would be good.